src/qt/walletmodel.cpp

```diff
@@ -704,10 +704,10 @@ bool WalletModel::hdEnabled() const
 
 int WalletModel::getDefaultConfirmTarget() const
 {
-    return nTxConfirmTarget;
+    return FIXME_IMPLEMENT_IPC_VALUE(nTxConfirmTarget);
 }
 
 bool WalletModel::getDefaultWalletRbf() const
 {
-    return fWalletRbf;
```

Comment:
Yep I guess most of these calls should be turned into async calls and not wait on a response synchronously blocking the GUI. Not necessarily in the first iteration of this, of course.